models/comments.js

```diff
@@ -1,5 +1,22 @@
 const db = require("../db/connection");
 
+exports.selectAllComments = () => {
+  return db.query(`SELECT * FROM comments ;`).then(({ rows }) => {
+    return rows.length;
+  });
+};
+
+exports.removeCommentById = (id) => {
+  return db
+    .query(`DELETE FROM comments WHERE comment_id = $1 RETURNING * `, [id])
+    .then(({ rows }) => {
+      if (rows.length === 0) {
+        return Promise.reject({ status: 404, msg: "invalid content" });
```

Comment:
The msg should say something like 'comment not found' instead - the content (comment_id) is valid, just not found in your db. That's also what a 404 is, so your status and msg should agree